api/s2n.h

```diff
@@ -141,6 +141,7 @@ extern int s2n_config_set_max_cert_chain_depth(struct s2n_config *config, uint16
 extern int s2n_config_add_dhparams(struct s2n_config *config, const char *dhparams_pem);
 extern int s2n_config_set_cipher_preferences(struct s2n_config *config, const char *version);
 extern int s2n_config_set_signature_preferences(struct s2n_config *config, const char *version);
+extern int s2n_config_set_ecc_preferences(struct s2n_config *config, const char *version);
```

Comment:
Can we update the README as with this new API?

Reply:
similarly to `s2n_connection_set_cipher_preferences`, please add `s2n_connection_set_ecc_preferences` if it's a straight forward change.